lib/test.js

```diff
@@ -58,8 +58,16 @@ function configure(sinon, config) {
             } catch (e) {
                 exception = e;
             }
-
-            if (typeof oldDone !== "function") {
```

Comment:
Oh, dear. We actually extracted `sinon.test` to this package in the hopes of someone volunteering to maintain it, as neither of us had much interest in actually maintaining it, as the core of `sinon` takes more than enough time in itself ... 

I will need to dig a little bit into the code here before I can give you a reply. But I _will_ do it :-)

P.S. I'm adding a little note to the README advertising for maintainers to step up, so if someone feels the urge ...